canonical/ubuntu.com
# Create dashboard page

## Done

- Create /pro/dashboard page

## QA

- https://ubuntu-com-12170.demos.haus/pro shows the pro page
- https://ubuntu-com-12170.demos.haus/pro/dashboard shows the pro dashboard page if logged in
- Check the subnav links. Should be one for /pro and one for the dashboard.
- Check the dropdown nav links. UA subscriptions link takes you to dashboard.

## Issue / Card

Fixes #https://github.com/canonical/commercial-squad/issues/733
